pkg/gui/custom_commands.go

```diff
@@ -146,13 +156,26 @@ func (gui *Gui) GenerateMenuCandidates(commandOutput string, filter string, form
 				}
 			}
 		}
-		err = temp.Execute(buff, tmplData)
+		err = tempItem.Execute(buffItem, tmplData)
+		if err != nil {
+			return candidates, gui.surfaceError(err)
+		}
+		err = tempDescr.Execute(buffDescr, tmplData)
 		if err != nil {
 			return candidates, gui.surfaceError(err)
 		}
```

Comment:
In some comment below i suggested to only use the label in the output
In that case we should fallback to the value of the label is not set.
```suggestion
		if labelFormat != "" {
			err = tempDescr.Execute(buffDescr, tmplData)
			if err != nil {
				return candidates, gui.surfaceError(err)
			}
		} else {
			buffDescr.Write(buffItem.Bytes())
		}
```
note that we also need to reflect this change in the docs

Reply:
Done! I agree, it's better if we only show the label; I don't know what I was thinking ...
Thanks for your understanding and help!